test/test_va_api_fixture.cpp

```diff
@@ -113,8 +120,46 @@ void VAAPIFixture::doGetMaxEntrypoints()
 void VAAPIFixture::doGetMaxNumConfigAttribs()
 {
     m_maxConfigAttributes = vaMaxNumConfigAttributes(m_vaDisplay);
+    ASSERT_GT(m_maxConfigAttributes, 0);
+}
 
-    EXPECT_EQ(m_maxConfigAttributes, 32);
+void VAAPIFixture::doGetMaxNumImageFormats()
+{
+    m_maxImageFormat = vaMaxNumImageFormats(m_vaDisplay);
+    ASSERT_GT(m_maxImageFormat, 0);
+}
+
+void VAAPIFixture::doGetMaxNumSubPicImageFormats()
+{
+    m_maxSubPicImageFormat =  vaMaxNumSubpictureFormats(m_vaDisplay);
+    ASSERT_GT(m_maxSubPicImageFormat, 0);
+}
+
+void VAAPIFixture::doQueryImageFormats()
+{
+    m_imageFmtList.resize(m_maxImageFormat);
+
+    ASSERT_STATUS(
```

Comment:
EXPECT_STATUS is probably better so we fall-through to next check below... this would allow additional information to be logged when the EXPECT fails.

Reply:
Updated.